server/parsec/components/memory/datamodel.py

```diff
@@ -289,8 +289,12 @@ class MemoryInvitation:
     type: InvitationType
     created_by_user_id: UserID
     created_by_device_id: DeviceID
+
```

Comment:
```suggestion
```

Reply:
I grouped the attribute on purpose, to easily visualize those that are type dependant